.github/workflows/ci-join.yaml

```diff
@@ -27,20 +27,39 @@ jobs:
         with:
           go-version: 'stable'
 
-      - name: Run join test
+      - name: Run Join Network Test - Full Sync Node
         run: |
           cd scripts/join
           sudo go test . -v \
             --integration \
             --timeout=0 \
-            --logs_file=docker_logs.txt \
+            --logs_file=docker_logs_full_sync.txt \
             --halo_tag="main" \
             --network="${{github.event.inputs.network || 'mainnet'}}"
 
-      - name: Upload docker logs
+      - name: Upload Docker Logs - Full Sync Node
         uses: actions/upload-artifact@v4
         if: always()
         with:
           name: docker-logs
-          path: scripts/join/docker_logs.txt
+          path: scripts/join/docker_logs_full_sync.txt
+          retention-days: 3
+
+      - name: Run Join Network Test - Snapshot Sync Node
```

Comment:
we also need to run this test periodically, for multiple networks preferrably

Reply:
Currently we support a manual choice between `omega` and `mainnet` and `mainnet` as the sane default. As far as I understand your idea is no make separate jobs for `omega`, `mainnet` and possibly `staging`?